src/libpgmoneta/backup.c

```diff
@@ -130,12 +137,25 @@ pgmoneta_backup(int client_fd, int server, uint8_t compression, uint8_t encrypti
    current = workflow;
    while (current != NULL)
    {
+      if (current->name)
```

Comment:
Just calculate and set in the `wf_` - you can use a process local variable with init in `setup()` and set in `teardown()`

Reply:
@jesperpedersen Wouldn't it be better if we add the framework to measure the time of _any_ workflow? That is why I decided to add the `name` field...
Also, if I measure the time between `setup()` and `teardown()`, that will be inaccurate right? Because all workflows first run `setup()` then `execute()` and then `teardown()`.
May I also ask the thought behind this design choice. Why is not that one workflow completes first(setup->execute->teardown) and then only the next workflow starts?

Reply:
For now, we just need the timing part - having a general solution integrated is planned

Reply:
Alright.

And I think I should only measure time of `execute()`, for reason I stated above ?

Reply:
Yes, that is ok